rebar.config

```diff
@@ -22,8 +22,8 @@
   {".*", "priv/lib/mongoose_mam_id.so", ["c_src/mongoose_mam_id.cpp"], [{env, [{"CXXFLAGS", "$CXXFLAGS -std=c++11"}]}]}
  ]}.
 
-%% We use functions introduced in Erlang/OTP 19
-{require_min_otp_vsn, "20"}.
+%% We use syntax introduced in Erlang/OTP 21
```

Comment:
We (will) use both functions and syntax introduced in Erlang/OTP 21, like logger or persistent_term, is just that for now we have only updated the stacktrace thing.

Reply:
hmm, I would rather remove the comment completely instead of manainig it as the option is rather self-explanatory :)

Reply:
Great as well. Btw, you could have pushed with [skip ci] or so, now you just spend a dollar on CircleCI :stuck_out_tongue: